data_steward/cdr_cleaner/cleaning_rules/vehicular_accident_concept_suppression.py

```diff
@@ -0,0 +1,151 @@
+"""
+Suppress Vehicular accidents related SNOMED concepts descendants
+
+Original Issues: DC-1959, DC-2212
+"""
+
+# Python imports
+import logging
+
+# Project imports
+from cdr_cleaner.cleaning_rules.deid.concept_suppression import AbstractBqLookupTableConceptSuppression
+from constants.cdr_cleaner import clean_cdr as cdr_consts
+from common import JINJA_ENV, CONDITION_OCCURRENCE
+from utils import pipeline_logging
+
+# Third party imports
+from google.cloud.exceptions import GoogleCloudError
+
+LOGGER = logging.getLogger(__name__)
+
+SUPPRESSION_RULE_CONCEPT_TABLE = 'vehicular_accident_descendant_concepts'
+
+VEHICULAR_ACCIDENTS_CONCEPT_QUERY = JINJA_ENV.from_string("""
+CREATE OR REPLACE TABLE `{{project_id}}.{{sandbox_id}}.{{concept_suppression_lookup_table}}` AS
```

Comment:
I think this works as is, but the Create table docs seem to suggest enclosing the query in ().